saiplayer/SaiPlayer.cpp

```diff
@@ -1022,309 +1049,338 @@ std::vector<std::string> SaiPlayer::tokenize(
     return tokens;
 }
 
-sai_status_t SaiPlayer::handle_bulk_route(
+sai_status_t SaiPlayer::handle_bulk_entry(
         _In_ const std::vector<std::string> &object_ids,
+        _In_ sai_object_type_t object_type,
         _In_ sai_common_api_t api,
         _In_ const std::vector<std::shared_ptr<SaiAttributeList>> &attributes,
-        _In_ const std::vector<sai_status_t>& expectedStatuses)
+        _Out_ std::vector<sai_status_t>& statuses)
 {
     SWSS_LOG_ENTER();
 
-    std::vector<sai_route_entry_t> routes;
+    sai_bulk_op_error_mode_t mode = SAI_BULK_OP_ERROR_MODE_IGNORE_ERROR; // TODO we need to get that from recording
+
+    uint32_t object_count = (uint32_t) object_ids.size();
 
-    for (size_t i = 0; i < object_ids.size(); ++i)
+    sai_status_t status = SAI_STATUS_SUCCESS;
+
+    if (api == SAI_COMMON_API_BULK_CREATE)
     {
-        sai_route_entry_t route_entry;
-        sai_deserialize_route_entry(object_ids[i], route_entry);
+        std::vector<uint32_t> attr_counts(object_count);
+        std::vector<const sai_attribute_t*> attr_lists(object_count);
 
-        route_entry.vr_id = translate_local_to_redis(route_entry.vr_id);
+        for (uint32_t idx = 0; idx < object_count; idx++)
+        {
+            attr_counts[idx] = attributes[idx]->get_attr_count();
+            attr_lists[idx] = attributes[idx]->get_attr_list();
+        }
```

Comment:
github diff is making this such a mess i'm not able to review it, every couple lines it switches from one to another file
could you move your function to some other place in SaiPlayer.cpp, like 200-300 lines above/below, so diff will not pick this up ? its really hard to view if lines are alternating so many times :( 
diff gets confused because of  { }
![image](https://user-images.githubusercontent.com/17052279/96627424-215c4900-1311-11eb-99cc-b418d83f7d7d.png)